### PR TITLE
Style/Update util.js to showing Korean language(80%)

### PR DESCRIPTION
**Pull request by ihan1004**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by fuksman** [comment]
Hello @ihan1004, nice work!
Please also enable the language in `src/json/constant.json`, you can use https://github.com/anyproto/anytype-ts/pull/585/files as a reference.

**Comment by ihan1004** [comment]
Thanks! I also enabled the language in `src/json/constant.json`.
Please re-check :)

File: src/json/constant.json [hunk 1]
```diff
@@ -560,7 +561,7 @@
 		"id": "Bahasa Indonesia",
 		"it": "Italiano",
 		"it-IT": "Italiano (Italia)",
-		"ko": "한국어",
```
Comment (fuksman) [inline src/json/constant.json:563]: Please revert this change, it's responsible for spellchecker and not for interface language.

**Review by fuksman — APPROVED** [review]

**Comment by fuksman** [comment]
@any contributor @ihan1004 l10n
